woocommerce/woocommerce-gateway-stripe
# 'Remove method' confirmation modal

Fixes #1876

### Description
In the Stripe settings page `http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe`, checking and unchecking any payment method reflects immediately on click. The changes in this PR introduces a confirmation modal for unchecking a selected method. 

![confirmation modal](https://user-images.githubusercontent.com/33387139/135064783-f2358c74-e775-4160-ad8f-c544f8a06ea8.gif)


# Testing instructions
- Enable `_wcstripe_feature_upe_settings`
- Enable UPE
- Go to `http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe` and uncheck any checked method. 
- Confirmation modal should open. On clicking `Cancel` should do nothing, on clicking `Remove` should uncheck the payment method.
